challenges/desafio3.js

```diff
@@ -0,0 +1 @@
+db.voos.find({"empresa.nome": "AZUL"}).count();
```

Comment:
Sabia que dá pra usar apenas o count, sem o find? Aí fica assim: 
```suggestion
db.voos.count({"empresa.nome": "AZUL"});
```